warehouse/core/src/main/java/datawave/mr/bulk/BulkIteratorEnvironment.java

```diff
@@ -69,4 +71,14 @@ public SortedKeyValueIterator<Key,Value> reserveMapFileReader(String mapFileName
         return null;
     }
 
+    @Override
+    public ServiceEnvironment getServiceEnv() {
```

Comment:
I wouldn't bother overriding this as it's deprecated in Accumulo 2+. If we don't, does something break?